### PR TITLE
Docs: Update the list of available blocks in Block Theme Overview

### PR DESCRIPTION
**Pull request by audrasjb**
## Description
This PR aims to update the list of available blocks in the "Block Theme Overview" documentation page.

## How has this been tested?
N/A

## Screenshots 
N/A

## Types of changes
Documentation update for WP 5.9.

### CODE REVIEW

File: docs/how-to-guides/themes/block-theme-overview.md [hunk 1]
```diff
@@ -119,6 +126,8 @@ As we're still early in the process, the number of blocks specifically dedicated
 -   Post Featured Image
 -   Post Hierarchical Terms
```
Comment (carolinan) [inline docs/how-to-guides/themes/block-theme-overview.md]: Post Hierarchical Terms was replaced with Post Categories

Both post categories and post tags are variations of the [post term block](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/post-terms), but the post term is not surfaced in the editor, the variations are.
-Which block is correct to mention here?

File: docs/how-to-guides/themes/block-theme-overview.md [hunk 2]
```diff
@@ -98,13 +98,20 @@ Note that it won't take precedence over any of your theme's templates with highe
...
+As we're still early in the process, the number of blocks specifically dedicated to these block templates is relatively small but more will be added as we move forward with the project. As of Gutenberg 11.9, the following blocks are currently available:
 
 -   Site Title
+-   Site Logo
+-   Site Tagline
 -   Template Part
 -   Query
 -   Query Loop
 -   Query Pagination
+-   Header
```
Comment (carolinan) [inline docs/how-to-guides/themes/block-theme-overview.md]: Header and footer are not new blocks, they are template parts, I guess we could call them aliases.

**Comment by carolinan** [comment]
Pattern block is added in 11.9.
https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/pattern

File: docs/how-to-guides/themes/block-theme-overview.md (hunk 1 above)
Comment (audrasjb) [reply docs/how-to-guides/themes/block-theme-overview.md]: I think it's better to mention what's actually available in the editor, so Post Categories would be our best choice

Comment (audrasjb) [reply docs/how-to-guides/themes/block-theme-overview.md]: Fixed in the last commit

File: docs/how-to-guides/themes/block-theme-overview.md (hunk 2 above)
Comment (audrasjb) [reply docs/how-to-guides/themes/block-theme-overview.md]: New proposal in the last commit

**Review by carolinan — APPROVED** [review]

**Review by aristath — APPROVED** [review]

**Comment by aristath** [comment]
The structure of these docs has changed so I believe this one is no longer relevant... I was trying to rebase the PR but it looks like the list of blocks no longer exists the way it used to

**Comment by t-hamano** [comment]
The content this PR is trying to update appears to have been removed as duplicate content by #38800.

@carolinan @aristath 

Can we close this PR?

**Pull request closed without merging**